components/06-examples/machtigen.hbs

```diff
@@ -0,0 +1,78 @@
+{{> '@header' }}
+<main>
+  <div>
```

Comment:
Ik weet wel waarom deze div er zit, maar ik stel voor deze div niet te plaatsen en het probleem bij de oorzaak op te lossen. Als dit soort tijdelijke oplossingen worden gemerged krijgen deze de ongewenste status "ok, dit is een goede oplossing"